app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -47,14 +59,26 @@ export const ListingDetailsFeatures = ({ listing, imageSrc }: ListingDetailsFeat
 
           <Description term={t("listings.features.unitFeatures")} description={""} />
         </dl>
-        {/* Waiting on new prop uptake from DAH-1133
+        {isRental(listing) && (
           <AdditionalFees
-          depositMin={listing.Deposit_Min?.toLocaleString()}
-          depositMax={listing.Deposit_Max?.toLocaleString()}
-          applicationFee={listing.Fee?.toLocaleString()}
-          costsNotIncluded={listing.Costs_Not_Included}
-          depositHelperText={"or one month's rent"}
-        /> */}
+            deposit={getDepositString(
+              listing.Deposit_Min?.toLocaleString(),
+              listing.Deposit_Max?.toLocaleString()
+            )}
+            applicationFee={listing.Fee ? `$${listing.Fee?.toLocaleString()}` : null}
```

Comment:
I noticed the angular app defaults to two decimal places for the fee (but not the deposit costs); my instinct is to say it doesn't matter, though it might be a good safety measure to do a `.toFixed(2)` in case of any wild data
| Angular | React |
| --- | --- |
![Screen Shot 2022-05-23 at 5 03 06 PM](https://user-images.githubusercontent.com/3513225/169924730-57060ef1-12b7-4fb3-8ff7-40d6c0635a4f.png) | <img width="553" alt="Screen Shot 2022-05-23 at 5 03 14 PM" src="https://user-images.githubusercontent.com/3513225/169925061-1c2b50f5-1f7b-4f3d-a966-2b3d82e7ea4f.png">